policyengine_us/variables/gov/states/mt/tax/income/subtractions/mt_disability_income_subtraction_person.py

```diff
@@ -0,0 +1,32 @@
+from policyengine_us.model_api import *
+
+
+class mt_disability_income_subtraction_person(Variable):
+    value_type = float
+    entity = Person
+    label = "Montana disability income subtraction at person level"
+    defined_for = StateCode.MT
+    unit = USD
+    definition_period = YEAR
+    reference = "https://rules.mt.gov/gateway/RuleNo.asp?RN=42%2E15%2E217"
+
+    def formula(person, period, parameters):
+        # first select head and spouse with ages below the specific threshold, and
+        # select those who are retired on total disability and disabled, and
+        # calculate the corresbonding disability benefits.
+        p = parameters(
+            period
+        ).gov.states.mt.tax.income.subtractions.disability_income
+        age_eligible = person("age", period) < p.age_threshold
```

Comment:
Is there a [cap](https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2022/12/Form-2-2022-Instructions.pdf#page=31)?

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/967b9dc6-7b19-457e-ac0e-ca0636e2b83e)